src/ace/managers/timer.c

```diff
@@ -162,5 +171,5 @@ void timerWaitUs(UWORD uwUsCnt) {
 	// timerGetPrec(): One tick equals: PAL - 0.40us, NTSC - 0.45us
 	ULONG ulStart = timerGetPrec();
 	UWORD uwTickCnt = uwUsCnt*2/5;
-	while(timerGetPrec() - ulStart < uwTickCnt);
+	while(timerGetPrec() - ulStart < uwTickCnt) {}
```

Comment:
Good example for clangformat. Guess adding something to empty code blocks will show in diff, which we agreed to mitigate. Should be in new lines?

Reply:
I think that such change is significant enough to have it fully shown in diff. Also, the following code looks awful and doesn't look as done deliberately.

``` c
while(timerGetPrec() - ulStart < uwTickCnt) {
}
```